parsel/selector.py

```diff
@@ -595,7 +595,7 @@ def attrib(self) -> Dict[str, str]:
         return dict(self.root.attrib)
 
     def cleaned(
-        self: _SelectorType, cleaner: Union[str, Cleaner] = "auto"
+        self: _SelectorType, cleaner: Union[str, Cleaner] = "html"
```

Comment:
This seems like it might be an accidental change from conflict resolution.

Reply:
@Gallaecio this is not accidential change. the cleaner type 'auto' is not listed in the valid cleaner list. It raises exception. 
So, the two valid cleaner right now is 'html' and 'text' and 'html' is default.